classes/class.ilObjAdobeConnect.php

```diff
@@ -1681,13 +1572,8 @@ public function getFolderIdByLogin($externalLogin)
 	 */
 	public function getContentIconAttribute($sco_id)
 	{
-		$session = $this->xmlApi->getBreezeSession();
-		
 		$icons = array();
-		if($session != NULL && $this->xmlApi->login($this->adminLogin, $this->adminPass, $session))
-		{
-			$icons[] = $this->xmlApi->getContentIconAttribute($sco_id, $this->sco_id, $session);
-		}
+		$icons[] = $this->xmlApi->getContentIconAttribute($sco_id, $this->sco_id, null);
```

Comment:
This could result in a different bevaviour:

Before: If there was a auth. issue, `$icons` was an empty array
With the change: There is always exactly one element in the `$icons` array